lib/rules/require-each-key.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const AstNodeInfo = require('../helpers/ast-node-info');
+const Rule = require('./base');
+
+const ERROR_MESSAGE = '{{#each}} block requires a valid key value to avoid performance issues';
```

Comment:
The [documentation](https://api.emberjs.com/ember/release/classes/Ember.Templates.helpers/methods/each) refers to `{{#each}}` as "helper", not "block".

Reply:
Changed block to helper